### PR TITLE
fix: check is color set

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: src/table/utils/styling-utils.js [hunk 1]
```diff
@@ -54,7 +57,7 @@ export const getBaseStyling = (styleObj, objetName, theme) => {
   const baseStyle = {
     backgroundColor: theme.table.backgroundColor,
     fontFamily,
-    color: styleObj?.fontColor ? getColor(STYLING_DEFAULTS.FONT_COLOR, theme, styleObj.fontColor) : color,
+    color: isSet(styleObj?.fontColor) ? getColor(STYLING_DEFAULTS.FONT_COLOR, theme, styleObj.fontColor) : color,
```
Comment (haxxmaxx) [inline src/table/utils/styling-utils.js:60]: Is there a test on these two cases?

Comment (LiKang6688) [reply src/table/utils/styling-utils.js:60]: Good question. Do not think there is any. Let me add some.

Comment (LiKang6688) [reply src/table/utils/styling-utils.js:60]: Done

File: src/table/utils/__tests__/styling-utils.spec.js [hunk 2]
```diff
@@ -113,6 +113,29 @@ describe('styling-utils', () => {
...
     });
+    it('should return styling with fontSize and padding when the index for font color is -1 and color is null', () => {
+      styleObj.fontColor = { index: -1, color: null };
+      const resultStyling = getBaseStyling(styleObj, objetName, theme);
+      expect(resultStyling).toEqual({
+        backgroundColor: '#323232',
+        borderColor: '#D9D9D9',
+        borderStyle: 'solid',
+        fontSize: 12,
+        padding: '6px 12px',
```
Comment (haxxmaxx) [inline src/table/utils/__tests__/styling-utils.spec.js:124]: Is this the real case scenario? that you don't get a color set?

Comment (haxxmaxx) [reply src/table/utils/__tests__/styling-utils.spec.js:124]: only if there is something set on the theme?

Comment (LiKang6688) [reply src/table/utils/__tests__/styling-utils.spec.js:124]: You are right. When there is no color from Layout and color from the theme, you get color from the theme. 
In the test case, there is no color from either Layout or theme, so you don't get any color.

Comment (LiKang6688) [reply src/table/utils/__tests__/styling-utils.spec.js:124]: Adding a new test case addressing it.

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**